god-mode.el

```diff
@@ -461,6 +462,78 @@ be ignored by `god-execute-with-current-bindings'."
                        universal-argument
                        universal-argument-more)))
 
+(defcustom god-translate-key-for-description
```

Comment:
I think this custom var is redundant and should be removed, as `god-mode-describe-key` is an entirely separate function.
If someone doesn't want to use this feature, they would simply not use or bind the `god-mode-describe-key` function.

Reply:
Good point. One reason I wanted to keep it around is because `god-mode-describe-key` is a bit limited compared to the regular `describe-key` (can't describe mouse clicks, menu items, or key bindings outside of `god-mode`). But I think the fix for this is simply to specify it in the README: users should know that if they want to describe (eg) a menu item,  they can simply turn off `god-mode` temporarily.

I will fix this in the next merge